src/main/java/com/graphhopper/converter/core/Converter.java

```diff
@@ -16,7 +16,7 @@ public static GHEntry convertFromGisgraphyAddress(GisgraphyAddressEntry gisgraph
         GHEntry rsp = new GHEntry(null, null, gisgraphyEntry.getLat(),
                 gisgraphyEntry.getLng(), gisgraphyEntry.getDisplayName(), null,
                 gisgraphyEntry.getCountry(), gisgraphyEntry.getCity(),
-                gisgraphyEntry.getState(), gisgraphyEntry.getStreetName(),
+                gisgraphyEntry.getState(),  gisgraphyEntry.getAdm3Name(), gisgraphyEntry.getStreetName(),
```

Comment:
When thinking about this a bit more, I am not sure if we should do this. We convert Adm3Name to county, but maybe this is not true for other regions of the world? Gisgraphy does not return county, maybe we just return null then instead?

Reply:
Yes, admin3 is not always county. Do you plan to add county @gisgraphy ?